app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -109,7 +112,7 @@ enum class KeyboardLayout(val index: Int, val keyboardDefinition: KeyboardDefini
     FITypeSplit(59, KB_FI_TYPESPLIT),
     FRThumbKeyV1(8, KB_FR_THUMBKEY_V1),
     FRThumbKeyV2(33, KB_FR_THUMBKEY_V2),
-    FRMessageEase(28, KB_FR_THUMBKEY_V1),
+    FRMessageEase(28, KB_FR_MESSAGEEASE),
```

Comment:
Thx for getting this one. Eventually I'm going to put all these back in order by index, because people adding to this file need to see the last number anyway.